buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1261,6 +1283,8 @@ def create_job(JOB_NAME, SDK_VERSION, SPEC, downstreamJobType, id){
     params.put('VENDOR_CREDENTIALS_ID_DEFAULT', VENDOR_CREDENTIALS_ID_DEFAULT)
     params.put('jobType', downstreamJobType)
     params.put('DISCARDER_NUM_BUILDS', DISCARDER_NUM_BUILDS)
+    params.put('SCM_REPO', 'https://github.com/Adambrousseau/openj9.git')
```

Comment:
Hard-coded personal fork, please remove (set in Job template)

Reply:
As per my previous comment. Will remove once testing is done.